eoc_journal/utils.py

```diff
@@ -0,0 +1,8 @@
+"""EOC Journal XBlock - Utils"""
+
+
+def _(text):
+    """
+    Dummy `gettext` replacement to make string extraction tools scrape strings marked for translation.
+    """
+    return text
```

Comment:
@bdero Thanks for these links, I wasn't familiar with the i18n service. It will be very useful in the future, but I don't think we can use it translate xblock field names and descriptions, which is currently the only place where we use the `_` function. It looks like skytap xblock is also using a [dummy `_` function](https://github.com/open-craft/xblock-skytap/blob/6a6c93a9566b13049edb0d9064819802368afa0b/xblock_skytap/skytap.py#L30) for field names & descriptions.